search-app/src/cmr/search/results_handlers/atom_spatial_results_handler.clj

```diff
@@ -1,19 +1,19 @@
 (ns cmr.search.results-handlers.atom-spatial-results-handler
   "A helper for converting spatial shapes into ATOM results"
-  (:require [clojure.data.xml :as x]
-            [cmr.spatial.polygon :as poly]
-            [cmr.spatial.point :as p]
-            [cmr.spatial.mbr :as m]
-            [cmr.spatial.geodetic-ring :as gr]
-            [cmr.spatial.cartesian-ring :as cr]
-            [cmr.spatial.line-string :as l]
-            [clojure.string :as s]
-            [cmr.common.util :as u]))
+  (:require [clojure.data.xml :as xml]
+            [clojure.string :as string]
+            ;[cmr.spatial.polygon :as poly]
```

Comment:
why not remove, instead of commenting out?

Reply:
+1

Reply:
I was not sure if it needed to be a require or an import.